app/src/main/java/com/blueprint/featureA/mapper/FeatureMapper.kt

```diff
@@ -0,0 +1,21 @@
+package com.blueprint.featureA.mapper
+
+import com.blueprint.featureA.data.model.local.FeatureAEntity
+import com.blueprint.featureA.domain.model.FeatureA
+
+
+fun FeatureA.toEntity(): FeatureAEntity {
+  return FeatureAEntity(
```

Comment:
Sitna stvar, kad su ovakve situacije ja bih skratila na expression:
`fun FeatureAEntity.toDomain(): FeatureA =
FeatureA(
    string = this.string
  )`

Reply:
Da. Sto da ne